nixos/modules/services/web-apps/porn-vault/default-config.nix

```diff
@@ -0,0 +1,158 @@
+# See https://gitlab.com/porn-vault/porn-vault/-/blob/dev/config.example.json
```

Comment:
Any reason to not just use the JSON file itself and do a `lib.fromJSON` instead?